schema/equals_test.go

```diff
@@ -60,43 +60,51 @@ func TestEquals(t *testing.T) {
 	// The "copy known fields" section of these function is to break if folks
 	// add new fields without fixing the Equals function and this test.
 	funcs := []interface{}{
-		func(x Schema) bool {
+		func(x *Schema) bool {
```

Comment:
Can you make the copy for these? I'm not concerned about the error here.

Reply:
there you go :-)